core/os/os.h

```diff
@@ -254,6 +254,9 @@ class OS {
 	void set_stdout_enabled(bool p_enabled);
 	void set_stderr_enabled(bool p_enabled);
 
+	virtual String mb_to_string(int p_encoding, const PackedByteArray &p_array) const;
+	virtual PackedByteArray string_to_mb(int p_encoding, const String &p_string) const;
```

Comment:
I think it would be better to be explicit here, I had no clue what "mb" stood for before reading the Windows implementation of it.
```suggestion
	virtual String multibyte_to_string(int p_encoding, const PackedByteArray &p_array) const;
	virtual PackedByteArray string_to_multibyte(int p_encoding, const String &p_string) const;
```

Reply:
Renamed, for the reference not all encoding supported by this use multiple byte pre character, but that's what Windows call it - https://learn.microsoft.com/en-us/windows/win32/api/stringapiset/nf-stringapiset-multibytetowidechar